bindings/rust/lib.rs

```diff
@@ -49,6 +49,9 @@ pub const GRAMMAR: &str = include_str!("../../grammar.js");
 /// The syntax highlighting query for this language.
 pub const HIGHLIGHT_QUERY: &str = include_str!("../../queries/highlights.scm");
 
+/// The symbol tagging query for this language.
+pub const TAGGING_QUERY: &str = include_str!("../../queries/tags.scm");
```

Comment:
```suggestion
pub const TAGS_QUERY: &str = include_str!("../../queries/tags.scm");
```

Reply:
Interestingly these names are somewhat inconsistent between crates e.g.
- https://docs.rs/tree-sitter-swift/latest/tree_sitter_swift/constant.TAGS_QUERY.html
- https://docs.rs/tree-sitter-rust/latest/tree_sitter_rust/constant.TAGGING_QUERY.html
- https://docs.rs/tree-sitter-javascript/latest/tree_sitter_javascript/constant.TAGGING_QUERY.html
- ...

Happy to change this, but it'd be nice to have a somewhat consistent naming convention for these. If we where to decide on one, do you think it'd be worth changing them in all the tree-sitter owned crates?

Reply:
Let's stick with TAGS, I wasn't aware that rust and js used TAGGING, that likely predates when the default template switched to TAGS